formula/src/main/java/com/instacart/formula/internal/FormulaManagerImpl.kt

```diff
@@ -196,4 +195,19 @@ internal class FormulaManagerImpl<Input, State : Any, Output>(
             key = formula.key(input)
         )
     }
+
+    private fun getOrInitChildTransitionListener(): TransitionListener {
```

Comment:
Optimization to only initialize `TransitionListener` once and re-use it for all child formulas.